packages/element-lib/src/sidetree/resolve.js

```diff
@@ -17,8 +17,8 @@ module.exports = async (sidetree) => {
     let transactionTimeHash;
     try {
       const blockchainTime = await sidetree.blockchain.getBlockchainTime(lastTransactionTime);
-      // eslint-disable-next-line
-      transactionTimeHash = blockchainTime.transactionTimeHash;
+      // FIXME: why is this undefined?
```

Comment:
This can be undefined if you try and get a block that was just created, or does not exist.